draft-ietf-tls-esni.md

```diff
@@ -295,28 +295,28 @@ Clients MUST parse the extension list and check for unsupported mandatory
 extensions. If an unsupported mandatory extension is present, clients MUST
 ignore the `ECHConfig`.
 
-# The "encrypted_client_hello" extension {#encrypted-client-hello}
+# The "ech" Extension {#encrypted-client-hello}
```

Comment:
Very minor nitpick: Looks like most other TLS extensions are left unabbreviated. It does mean we lose the nice parallel between `ech_nonce` and `ech_outer_extension`, though if we pick up https://github.com/tlswg/draft-ietf-tls-esni/pull/292, the former will go away. Dunno how important a prefixed `ech_outer_extension` is.

Reply:
I'll revert to the old name and we can revisit extension naming once the dust settles.